src/main/java/Event.java

```diff
@@ -0,0 +1,21 @@
+public class Event extends Task {
+    protected final String sym = "E";
+    private String dayAndTime;
+
+    Event (String description, String dayAndTime) {
+        super(description);
+        this.dayAndTime = dayAndTime;
+    }
+
+    @Override
+    String getSym() {
+        return this.sym;
+    }
```

Comment:
May I check what is the purpose of this override? Same with deadline and todo classes. Seems redundant as the getSym() in Task is the same.